salt/metalk8s/containerd/installed.sls

```diff
@@ -22,3 +23,17 @@ Install containerd:
       - test: Repositories configured
       - pkg: Install runc
       - pkg: Install container-selinux
+
+Configure registry IP in containerd conf:
+  file.managed:
+    - name: /etc/containerd/config.toml
+    - makedirs: true
+    - contents: |
+        [plugins]
+          [plugins.cri]
+            [plugins.cri.registry]
+              [plugins.cri.registry.mirrors]
+                [plugins.cri.registry.mirrors."{{ defaults.registry_ip }}"]
```

Comment:
I might be wrong, but this TOML can also be written:
```
[plugins.cri.registry.mirrors."{{ defaults.registry_ip }}"]
  endpoint = ...
```
, without the need for all sections before.

Reply:
Yes, it can be written like that, I just put it like it because it is the output of `containerd config`